cmd/homi/setup/cmd.go

```diff
@@ -299,11 +300,36 @@ func genValidatorKeystore(privKeys []*ecdsa.PrivateKey) {
 
 	for i, pk := range privKeys {
 		pwdStr := RandStringRunes(params.PasswordLength)
-		ks.ImportECDSA(pk, pwdStr)
+		account, _ := ks.ImportECDSA(pk, pwdStr)
```

Comment:
`ImportECDSA` function generates UTC-XXX file and the file path is stored in `account` variable. And so I read the file to get the address of reward.